cloud/linode/cilium_loadbalancers.go

```diff
@@ -481,6 +482,11 @@ func (l *loadbalancers) getCiliumLBIPPool(ctx context.Context, service *v1.Servi
 
 // NOTE: Cilium CRDs must be installed for this to work
 func (l *loadbalancers) ensureCiliumBGPPeeringPolicy(ctx context.Context) error {
+	if raw, ok := os.LookupEnv("BGPCustomIDMap"); ok {
+		if err := json.Unmarshal([]byte(raw), &regionIDMap); err != nil {
```

Comment:
Can we add info logging here that we're overriding the default region map if this variable is set?